src/app/shared/components/course-card/course-card.component.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../../../styles/main.scss';
```

Comment:
definitely use an absolute path here and in other places)

Reply:
done